.cirrus.yml

```
@@ -9,7 +9,7 @@ env:
   RUSTDOCFLAGS: -D warnings
   TOOL: cargo
   # The MSRV
-  TOOLCHAIN: 1.56.1
+  TOOLCHAIN: 1.63
```

Make this change and I don't think you'll need to reinstall the toolchain on line 159 below.
```suggestion
  TOOLCHAIN: 1.63.0
```

---

Sounds good, fixed.